### PR TITLE
feat(hardware-testing): DVT Gripper assembly QC test

### PR DESCRIPTION
**Pull request by andySigler**
# Overview

DVT gripper assembly qc script functional. Will need updates in a week or so for new force-gauge fixture, but this is working as-is.

This script tests:
- Gripper Z-axis currents/speeds, comparing estimated motor position to encoder
- Desired jaw force vs measured force using force gauge
- Desired jaw width vs measured jaw with using gauge blocks
- Capacitive probe values in air, with probe, and touching the deck

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

File: api/src/opentrons/hardware_control/ot3api.py [hunk 1]
```diff
@@ -1406,6 +1383,12 @@ async def hold_jaw_width(self, jaw_width_mm: int) -> None:
         await self._hold_jaw_width(jaw_width_mm)
         self._gripper_handler.set_jaw_state(GripperJawState.HOLDING_CLOSED)
 
+    @property
+    def jaw_width(self) -> Optional[float]:
```
Comment (sfoster1) [inline api/src/opentrons/hardware_control/ot3api.py]: This isn't really the right place for this API. We should expose this through something on the gripper handler that you'd access by doing `get_instruments()` or something, or maybe make sure we're returning it in `current_position()`. We generally avoid having functions like this that give you details about an instrument on the main object without an arguments.

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Glad for the tests, but let's split this up to get the functional changes to the hardware controller into a different PR. I think we also need to change the way that get-gripper-jaw api works; it really should be a function that takes a mount, or maybe an attribute of the return value of `get_instruments()`, or another axis that's in current_position or something - this kind of folding it into a function on the main object is not really the style of the rest of the API.

**Comment by andySigler** [comment]
@sfoster1 same as #12386, I'd like to leave this here for a day or two full test is settled IRL, then will make said changes before a merge 👍

**Review by sfoster1 — APPROVED** [review]
nice!

**Pull request merged**